### PR TITLE
Add a clobber command. r=mossop

### PR DESCRIPTION
**Pull request by jsantell**

### CODE REVIEW

**Comment by Mossop** [comment]
It won't work on windows but might as well take it anyway.

**Comment by victorporof** [comment]
Should maybe also remove the lib and build-config.json files?

**Comment by victorporof** [comment]
I also suggest having this written in python or node so that we may use it on windows. If written in js, can't import modules.

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "revision": "5cd1dc12f984"
   },
   "scripts": {
+    "clobber": "rm -r .electron; rm build-config.json; rm -r node_modules",
```
Comment (pdehaan) [inline package.json]: Nit: use something like [**rimraf**](https://www.npmjs.com/package/rimraf) for cross platform greatness.

Comment (victorporof) [inline package.json]: If you delete node_modules you can't use rimraf again.

**Comment by victorporof** [comment]
Ok, landing this, will add the `rm -rf lib`.

**Comment by victorporof** [comment]
https://github.com/mozilla/tofino/commit/12425675f8474763ff24da538fce06b2a2f74625

**Pull request closed without merging**